config/appConfig.js

```diff
@@ -13,11 +17,14 @@ export default {
     tokenUrl: process.env.OAUTH_TOKEN_URL || 'https://isso.nypl.org/oauth/token',
     loginUrl: process.env.OAUTH_LOGIN_URL || 'https://isso.nypl.org/auth/login',
     clientId: process.env.CLIENT_ID || 'platform_admin',
-    // clientSecret should be put in .env file
+    clientSecret: process.env.CLIENT_SECRET,
```

Comment:
So I guess we should just try to keep this consistent — I think the approach you initially took of keeping secrets totally out of the `appConfig` file makes sense, so I'd remove this here.

Reply:
The reason I put it back is so that we are able to only call `dotEnv.config()` once in `appConfig.js` and all the configuration we need will be all in this file, instead of making the components to call both environment variables and the values in `appConfig.js`.

I want to make `appConfig.js` a single source of all the configurations. If the information is not credential, it will be coded in it directly, otherwise, `appConfig.js` will get it from ` .env`

Reply:
yeah I just double checked. You have to call `dotEnv.config()` to access to environment variables.